packages/react-charts/src/components/Chart/Chart.tsx

```diff
@@ -460,7 +459,7 @@ export const Chart: React.FunctionComponent<ChartProps> = ({
     >
       {children}
       {getLegend()}
-    </VictoryChart>
+    </VictoryChartWithContainerComponent>
```

Comment:
Is this a required change (i.e., do we need to cast to `any`)?

There are more components that use containerComponent prop, other than Chart and ChartAxis, so curious if this is blocking?

Reply:
These are the only 2 that threw errors. I agree it'd be better to redefine `victory` types or update them upstream. We can open an issue for this.